app.js

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable  max-classes-per-file */
```

Comment:
**Optional**
1. Please consider breaking down the code into multiple files each uses one class for a cleaner code base and simplicity.
2. Therefore, you won't have to disable one crucial property from the ESlint.